debian/changelog

```diff
@@ -1,3 +1,9 @@
+history-service (0.3~1ubports1) xenial; urgency=medium
```

Comment:
Please dont do this - you are supposed to change the last digits, as the first pair of numbers is the upstream version no and should not be touched by us (though in this case it wouldnt matter but its a good habit)

Reply:
oh sorry, so 0.3~0ubports2 is good ? 
The ubports documentation say that we should keep the last digit if anything changed in debian control. Please advise me